lib/helper/Polly.js

```diff
@@ -65,15 +68,20 @@ class Polly extends Helper {
   }
 
   // Start mocking network requests/responses
-  async _startMocking(title = 'Test') {
-    if (!(this.helpers && this.helpers.Puppeteer)) {
-      throw new Error('Puppeteer is the only supported helper right now');
+  async startMocking(title = 'Test') {
```

Comment:
Requires documentation for this method, if it goes to public API. 
Please mention that `mockRequest` automatically starts request mocking

Reply:
Added doc. Is this proper?